### PR TITLE
issues/88

### PR DESCRIPTION
**Pull request by cahirodoherty-learningpool**
Display language based off _activeLanguage on initializeWrapper and set on onLanguageChanged

Part solution to [issues/88](https://github.com/adaptlearning/adapt-contrib-xapi/issues/88) in combination with https://github.com/adaptlearning/adapt-contrib-languagePicker/pull/46

### CODE REVIEW

**Comment by deltanetdan** [comment]
@cahirodoherty-learningpool which part of issues/88 does this fix?

**Review by eleanor-heath — APPROVED** [review]

**Comment by cahirodoherty-learningpool** [comment]
> @cahirodoherty-learningpool which part of issues/88 does this fix?

Hi Dan, this is tied to issue 88 in that it will allow xAPI to set the active language and the language picker will pick this up on return to the course. The work on language picker also allows for a param to be passed to set this language on first run if desired

**Comment by deltanetdan** [comment]
Thanks Cahir, do I need to change any config? I tested with latest FW and both xAPI and language picker issues/88 branches and still get the language picker menu on return.  I've set xapi _shouldTrackState = true and language picker _showOnCourseLoad = true as per the issue.

**Comment by cahirodoherty-learningpool** [comment]
> Thanks Cahir, do I need to change any config? I tested with latest FW and both xAPI and language picker issues/88 branches and still get the language picker menu on return. I've set xapi _shouldTrackState = true and language picker _showOnCourseLoad = true as per the issue.
> 
> Note: testing on SCORM cloud.

Hi Dan, you're right. I need to take another look at this

File: js/adapt-contrib-xapi.js [hunk 1]
```diff
@@ -279,6 +279,8 @@ define([
       // Update the language.
       this.set({ displayLang: newLanguage });
 
+      Adapt.offlineStorage.set('_activeLanguage', newLanguage);
```
Comment (moloko) [inline js/adapt-contrib-xapi.js:282]:
```suggestion
      Adapt.offlineStorage.set('lang', newLanguage);
```

**Comment by oliverfoster** [comment]
closing as stale and in conflict

**Pull request closed without merging**